Description: Fix licensing issues. This patch is in head, please see: http://svn.php.net/viewvc?view=revision&revision=317358
Author: Mathias Ertl <apt-repository@fsinf.at>

Index: php-phpdocumentor-1.4.3/PhpDocumentor-1.4.3/phpDocumentor/Converters/XML/DocBook/peardoc2/Plain.php
===================================================================
--- php-phpdocumentor-1.4.3.orig/PhpDocumentor-1.4.3/phpDocumentor/Converters/XML/DocBook/peardoc2/Plain.php	1970-01-01 09:13:08.000000000 +0000
+++ php-phpdocumentor-1.4.3/PhpDocumentor-1.4.3/phpDocumentor/Converters/XML/DocBook/peardoc2/Plain.php	2011-11-17 17:40:59.739105752 +0000
@@ -1,26 +1,32 @@
 <?PHP
 /* vim: set expandtab tabstop=4 shiftwidth=4: */
-// +----------------------------------------------------------------------+
-// | PHP Version 4                                                        |
-// +----------------------------------------------------------------------+
-// | Copyright (c) 1997-2002 The PHP Group                                |
-// +----------------------------------------------------------------------+
-// | This source file is subject to version 2.0 of the PHP license,       |
-// | that is bundled with this package in the file LICENSE, and is        |
-// | available at through the world-wide-web at                           |
-// | http://www.php.net/license/2_02.txt.                                 |
-// | If you did not receive a copy of the PHP license and are unable to   |
-// | obtain it through the world-wide-web, please send a note to          |
-// | license@php.net so we can mail you a copy immediately.               |
-// +----------------------------------------------------------------------+
-// | Authors: Stephan Schmidt <schst@php.net>                             |
-// +----------------------------------------------------------------------+
 
 /**
  * XML/Beautifier/Renderer/Plain.php
+ * phpDocumentor :: automatic documentation generator
+ *
+ * PHP versions 4 and 5
+ *
+ * LICENSE:
+ *
+ * This library is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU Lesser General
+ * Public License as published by the Free Software Foundation;
+ * either version 2.1 of the License, or (at your option) any
+ * later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
  *
  * @package  XML_Beautifier
  * @author   Stephan Schmidt <schst@php.net>
+ * @license  http://www.opensource.org/licenses/lgpl-license.php LGPL
  */
 
 /**
@@ -247,4 +253,4 @@
         return $xml;
     }
 }
-?>
\ No newline at end of file
+?>
